catalog/resource_quality_monitor.go

```diff
@@ -90,7 +90,18 @@ func ResourceQualityMonitor() common.Resource {
 				return err
 
 			}
-			return common.StructToData(endpoint, monitorSchema, d)
+			oldWarehouseId := d.Get("warehouse_id").(string)
+			oldSkipBuiltinDashboard := d.Get("skip_builtin_dashboard").(bool)
```

Comment:
these fields only available for create, not update - should we flag this in the doc as well?

Reply:
yes, good idea.